server/odc-migrate/src/main/resources/migrate/common/V_4_3_1_4__add_logical_database.sql

```diff
@@ -0,0 +1,57 @@
+alter table connect_database modify column connection_id bigint(20) DEFAULT NULL COMMENT 'refernce to connect_connection.id';
+alter table connect_database modify column connection_id bigint(20) DEFAULT NULL COMMENT 'refernce to connect_connection.id';
+alter table connect_database modify column connection_id bigint(20) DEFAULT NULL COMMENT 'refernce to connect_connection.id';
+alter table connect_database add column `type` varchar(32)  NOT NULL DEFAULT 'PHYSICAL' COMMENT 'optional value: PHYSICAL, LOGICAL';
+alter table connect_database add column `alias` varchar(256) DEFAULT NULL COMMENT 'alias name for database';
+alter table `connect_database` modify `last_sync_time` datetime default null comment 'last synchronizing time';
+
+
+create table if not exists `connect_logical_db_meta`(
+    `id` bigint(20) NOT NULL AUTO_INCREMENT,
+    `create_time` datetime NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    `update_time` datetime DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
+    `dialect_type` varchar(64) NOT NULL COMMENT 'physical database dialect type',
+    `environment_id` bigint(20) NOT NULL COMMENT 'reference to collaboration_environment.id',
+    `connect_database_id` bigint(20) NOT NULL COMMENT 'reference to connect_database.id whose type is LOGICAL',
+    `organization_id` bigint(20) NOT NULL COMMENT 'reference to iam_organization.id',
+    PRIMARY KEY (`id`),
+    UNIQUE KEY `uk_connect_logical_db_meta_db_id` (`connect_database_id`)
+);
+
+create table if not exists `connect_logical_db_physical_db`(
```

Comment:
if maintain logical database in connect_database , here may connect_database_mapping, as it is assocated in same table

Reply:
done~